keyboards/bear_face/rules.mk

```diff
@@ -2,20 +2,12 @@
 MCU = atmega32u4
```

Comment:
Please add `DEFAULT_FOLDER = bear_face/v1` to the end of this file.

Reply:
fixed